index.html

```diff
@@ -302,17 +301,22 @@ <h1 class="contat">Contact me</h1>
                             a project that needs coding. I’d love to help with it </p>
                     </div>
 
-                    <div class="form">
-                        <input type="text" id="fname" class="form1" name="firstname" placeholder="Yeremias">
-                        <input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com">
-                        <textarea id="subject" name="subject" class="form3" placeholder="Write your message here"
-                            style="height:90px"></textarea>
-
-                    </div>
+                    <form class="form1" action="https://formspree.io/f/xgeqypdd" method="post">
+                        <ul class="widgets">
+                            <li><input type="text" id="fname" class="form1" name="firstname" maxlength="30"
+                                    placeholder="Abdimulhin" required></li>
+                            <li><input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com"
+                                    required></li>
+                            <li><textarea id="subject" name="subject" class="form3"
+                                    placeholder="Write your message here" maxlength="500" required
+                                    style="height:90px"></textarea></li>
+
+                            <li class="button">
+                                <button class="see" type="submit">Get In Touch</button>
+                            </li>
+                        </ul>
+                    </form>
 
-                    <div class="form-button">
-                        <button class="see">Get In Touch</button>
-                    </div>
                     <br>
 
 
```

Comment:
- [ ] Your whole contact section should be wrapped with a footer tag to follow the [HTML semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) due to SEO purpose :blush: